helm_deploy/values-preprod.yaml

```diff
@@ -26,7 +26,7 @@ env:
   JAVA_OPTS: "-Xmx2048m"
   HMPPS_AUTH_BASE_URI: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
   HMPPS_AUTH_REDIRECT_BASE_URI: "https://calculate-journey-variable-payments-preprod.apps.live-1.cloud-platform.service.justice.gov.uk"
-  CRON_AUTOMATIC_LOCATION_MAPPING: "-"
+  CRON_AUTOMATIC_LOCATION_MAPPING: "0 30 3 * * ?"
```

Comment:
Runs at 3:30 am.  FYI this has no other dependency on anything else running in the morning.